tests/ui/explicit_deref_methods.stderr

```diff
@@ -1,67 +1,67 @@
-error: explicit deref method call
-  --> $DIR/dereference.rs:30:19
+error: explicit `deref` method call
+  --> $DIR/explicit_deref_methods.rs:30:19
    |
 LL |     let b: &str = a.deref();
    |                   ^^^^^^^^^ help: try this: `&*a`
    |
    = note: `-D clippy::explicit-deref-methods` implied by `-D warnings`
 
-error: explicit deref_mut method call
-  --> $DIR/dereference.rs:32:23
+error: explicit `deref` method call
```

Comment:
This loses the `_mut` prefix. Is that intentional?

Reply:
I didn't keep track of which one was called. I could for the last call in the chain, but that still leaves the mixed case to handle. e.g. `x.deref_mut().deref()`

Reply:
The last call is now tracked.